### PR TITLE
7a - nim project create - for issue #862

### PR DESCRIPTION
**Pull request by JamieDawson**
Here is a document for `7a. nim project create`.

7a nim project create

### CODE REVIEW

**Review by rabbah — APPROVED** [review]

File: docs/key-value-storage.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  - The `python` SDK
+- Common Redis commands
+
+### Conceptual Overview
+
+A KV behaves like a database where you can create keys, apply values to those keys, and then call on them to extract the data that you need.
+
+In this visual example, you can store user info, such as their City, State, and Country, and apply the correct values to them. This allows you to keep track of data and request the data when needed.
+
+![](https://github.com/JamieDawson/docs/blob/10b-two-types-storage-kv/static/img/kv-store-redis-example.png)
```
Comment (rabbah) [inline docs/key-value-storage.md:45]: this is referring to your fork. instead use a relative link `./docs/...` so that it works for anyone's fork.

**Review by rabbah — COMMENTED** [review]
check references to images that are fully qualified and referring to your fork - please replace those with local/relative links.

**Comment by JamieDawson** [comment]
Issues fixed in this document:

1) Deleted `key-value-storage.md` file. That file isn't suppose to be in here.
2) Fixed broken images and replaced them with local ones.
3) Attached document to siebar.
4) `git rebased` all commits to one.

Pending Re-review.

**Review by rabbah — COMMENTED** [review]
The file  `static/img/7a-nim-project-create.png` is not used in this PR?

**Review by rabbah — COMMENTED** [review]
Also "7a" has no meaning in this repo. Should use a different naming convention (here just drop it.

**Comment by JamieDawson** [comment]
All changes applied.

**Comment by JamieDawson** [comment]
File that needs to be checked:
Getting started --->  Creating your project

File: sidebars.js [hunk 2]
```diff
@@ -32,9 +32,10 @@ module.exports = {
 			'workbench-playground-introduction',
 			'workbench-playground-exports',
 		],
-		'Getting Started': ['getting-started', 'building'],
+		'Getting Started': ['getting-started', 'building', 'nim-creating-project'],
 		'Developing and Deploying Serverless APIs': [
 			'sample-project-walkthrough',
+			'stateful-api-examples',
```
Comment (rabbah) [inline sidebars.js:38]:
```suggestion
```

**Comment by rabbah** [comment]
@JamieDawson I merged this PR but this one is relevant to the discussion in #30 as the reference docs for projects is currently not linked to this "how to" but should be so that a reader can dig into the details they will eventually need.

**Pull request merged**